build.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+setupJava(){
+    export JAVA_HOME=`type -p javac|xargs readlink -f|xargs dirname|xargs dirname|sed 's/8/11/'`
+    export PATH=$PATH:$JAVA_HOME/bin/
+    export M2_HOME=/usr/share/maven/
+    export M2=$M2_HOME
+    export MAVEN_OPTS='-Xmx1048m -XX:MaxPermSize=512m -Xms256m'
```

Comment:
I think that one of these options (MaxPermSize) is no longer supported in Java 11. I believe that is one of the warnings generated initially. maybe remove -XX:MaxPermSize=512m

Reply:
https://github.com/archimatetool/archi/issues/153

you're talking about this?  I removed it.